clients/fides-js/src/components/Overlay.tsx

```diff
@@ -142,6 +141,14 @@ const Overlay: FunctionComponent<Props> = ({
     }
   }, [showBanner, cookie, options.debug, bannerIsOpen]);
 
+  useEffect(() => {
+    const cssUrl = new URL("fides.css", options.privacyCenterUrl);
+    const link = document.createElement("link");
+    link.rel = "stylesheet";
+    link.href = cssUrl.href;
+    document.head.appendChild(link);
+  }, [options.privacyCenterUrl]);
```

Comment:
actually, now that I'm thinking about this more, I think we actually want this css _in_ the `fides.js` bundle (that comes from the `fides-js` API endpoint). see [this comment](https://ethyca.atlassian.net/wiki/spaces/EN/pages/2774499406/2023-08-31+Customizing+fides.js+and+privacy+center+CSS?focusedCommentId=2775449606)

in other words, I think the flow would be something like...

1. user uploads some CSS which gets saved to the Fides server
2. the fides-js endpoint queries to see if there's any custom CSS
3. if there is, it stuff it into the fides.js payload. we used to do this with the fides.css file but not anymore. you can still find that code [here](https://github.com/ethyca/fides/commit/bbbf31982b7ec0119910c0500bc825007707c4d5) though (search for `fidesCss`)
4. then **in theory**, no changes are needed in fides.js. but I wouldn't be surprised if I'm wrong 😄 

that does mean there doesn't seem to be as much of a need for your new `fides-css` endpoint, except for perhaps that `refresh` logic. that said, the `fides-css` endpoint might be useful for the privacy center, which doesn't get bundled javascript